src/API/Entity.php

```diff
@@ -1407,14 +1402,7 @@ public function loadModelValues($model, $formData)
      */
     function simpleSave($model)
```

Comment:
I wonder if we even need this function anymore.  The only helpful thing it does it get the primary key of the recent save which is one line.  I think that with the rest of the recent changes, this function can be retired.